Sources/Preferences/PreferencesTabViewController.swift

```diff
@@ -93,6 +94,26 @@ final class PreferencesTabViewController: NSViewController, PreferencesStyleCont
 		}
 	}
 
+	private func updateWindowTitle(tabIndex: Int) {
+		self.window.title = {
+			if preferencePanes.count > 1 {
+				return preferencePanes[tabIndex].preferencePaneTitle
+			} else {
+				let appBundle = Bundle.main
+				let appName = appBundle.localizedInfoDictionary?["CFBundleDisplayName"] as? String
+					?? appBundle.infoDictionary?["CFBundleDisplayName"] as? String
+					?? appBundle.localizedInfoDictionary?["CFBundleExecutable"] as? String
+					?? appBundle.infoDictionary?["CFBundleExecutable"] as? String
```

Comment:
Can you make this an `Bundle` extension in util.swift? So it would just be `Bundle.main.name`.

Reply:
After checking `CFBundleExecutable`, I would also try to use the `Bundle#bundleIdentifier`, just in case everything else fails. Not likely, but doesn't hurt to be thorough. And as a last resort we could just use `"<Unknown>"` or something. I think that's better than defaulting to just `Preferences` without app name, as it doesn't make it clear to the user that something is wrong.

Reply:
In my own code, I have a helper that makes getting the name less verbose:

```swift
func getValue(_ key: String) -> String? {
	return object(forInfoDictionaryKey: key) as? String
}

// ...

getValue("CFBundleDisplayName") ??
getValue("CFBundleName") ??
```

Reply:
~Do you know if `object(forInfoDictionaryKey:)` uses the localized keys if available? 🤔~ Looks like it does, nice. What would you want me to be using `bundleIdentifier` for? Instead of the name, e.g. `"com.example.BestApp Preferences"`?